public/docs/_examples/server-communication/dart/.gitignore

```diff
@@ -0,0 +1,27 @@
+# See https://www.dartlang.org/tools/private-files.html
+
+# Files and directories created by pub
+.buildlog
```

Comment:
@zoechi I think it was created by github actually. I first created this sample in the [ng2-dart-samples](https://github.com/ng2-dart-samples/angular.io-server-communication) organization.